@ecomplus/widget-product/src/components/js/EcProduct.js

```diff
@@ -73,6 +73,14 @@ export default {
       return this.selectedVariation.name || name(this.body)
     },
 
+    selectedVariationQuantity () {
+      return (this.selectedVariation.quantity > 0 && this.selectedVariation.quantity < 13) ? this.selectedVariation.quantity : 0
+    },
+
+    almostOutOfStock () {
+      return (this.body.quantity > 0 && this.body.quantity < 13) ? this.body.quantity : 0
```

Comment:
Mas de onde c tirou este 13 @matheusgnreis ?

Reply:
Expliquei no debaixo. Com base nos sites que verifiquei e utilizam essa ferramenta

Reply:
Hmm, mas no mínimo isto deveria ser configurável, não? Uma prop talvez, aí o padrão pode ser 13 sim.
Com certeza isto varia de loja para loja, não acho que devemos forçar um número...